parser/match_operations_test.go

```diff
@@ -1408,6 +1408,132 @@ func TestMatchOperations(t *testing.T) {
 			matches: nil,
 			err:     true,
 		},
+		"nil amount ops (sub account optional - when sub-account exist)": {
+			operations: []*types.Operation{
+				{
+					Account: &types.AccountIdentifier{
```

Comment:
Can we add a group of operations in which one with sub account, another does not?

Reply:
updated this.